docs/src/main/sphinx/sql/create-view.rst

```diff
@@ -8,6 +8,7 @@ Synopsis
 .. code-block:: text
 
     CREATE [ OR REPLACE ] VIEW view_name
+    [ COMMENT view_comment ]
```

Comment:
It would be nice to have an example query with COMMENT as well

Reply:
Added an example.